tests/test_configuration_common.py

```diff
@@ -326,7 +326,13 @@ def test_config_common_kwargs_is_complete(self):
         missing_keys = [key for key in base_config.__dict__ if key not in config_common_kwargs]
         # If this part of the test fails, you have arguments to addin config_common_kwargs above.
         self.assertListEqual(
-            missing_keys, ["is_encoder_decoder", "_name_or_path", "_commit_hash", "transformers_version"]
+            missing_keys,
+            [
+                "is_encoder_decoder",
+                "_name_or_path",
+                "_commit_hash",
+                "transformers_version",
+            ],
```

Comment:
There is no reason to reformat this here. Please revert this change.

Reply:
Thanks for the comments @sgugger. I fixed the first. This specific reformat was enforced when running `make style`. If I revert it the code fails the quality check.

Reply:
@KMFODA it can be fixed by removing the trailing comma in `"transformers_version",` and then running `make style` :) Alternatively, you can copy/paste from the original.